packages/kbn-monaco/src/esql/lib/ast/definitions/options.ts

```diff
@@ -80,6 +80,7 @@ export const appendSeparatorOption: CommandOptionsDefinition = {
     params: [{ name: 'separator', type: 'string' }],
   },
   optional: true,
+  skipCommonValidation: true, // tell the validation engine to use only the validate function here
```

Comment:
Is there any reason we can't just use the presence of the `validate` method to decide (i.e. a clean override)?

Reply:
Initially that was the design, but then I rolled back to this other decision.
There are several reasons for this change:
* shared validation code is quite a big surface and replicate it every time to just add a tiny little bit of extra ad-hoc validation doesn't make sense
* therefore I want a way to add only this extra validation bit on top of the common one
* ideally I would like to have a "positive" flag (something like `commonValidation: true`) but I've only few cases (one so far) where this extra bit is required and adding a "negative" ("skipCommonValidation") was the less intrusive one